.gitignore

```diff
@@ -3,6 +3,8 @@ build
 node_modules
 .env
 .DS_Store
+.idea
 *~
 tag-message-*.txt
 release-*.html
+npm-debug.log
```

Comment:
small nit: this shouldn't be in the .gitignore: if there is an npm-debug.log in your dir, that npm error needs to be addressed. Pushing with an npm-debug.log file implies you've pushed code that has problems running.